File: app/src/main/java/com/jnjcomu/edison/factory/InterpolatorFactory.java
package com.jnjcomu.edison.factory;

import android.view.animation.Interpolator;

import com.jnjcomu.edison.ui.LogoInterpolator;

/**
 * @author CodeRi13 <ruto1924@gmail.com>
 * @since 2017-05-27
 */

public class InterpolatorFactory {
    public static Interpolator getDefaultLogoInterpolator() {
        return new LogoInterpolator(
                LogoInterpolator.DEFAULT_AMPLITUDE,
                LogoInterpolator.DEFAULT_FREQUENCY
        );
    }
}

File: app/src/main/java/com/jnjcomu/edison/factory/InterpolatorFactory.kt
package com.jnjcomu.edison.factory

import com.jnjcomu.edison.ui.LogoInterpolator

/**
 * @author CodeRi13 <ruto1924@gmail.com>
 * @since 2017-05-27
 */

object InterpolatorFactory {
    fun makeLogoInterpolator() = LogoInterpolator(
            LogoInterpolator.DEFAULT_AMPLITUDE,
            LogoInterpolator.DEFAULT_FREQUENCY
    )
}
